dashboard/src/components/logs/LogsHeader/LogsHeader.tsx

```diff
@@ -73,8 +75,23 @@ export default function LogsHeader({
     }
 
     onToDateChange(null);
+    setCurrentTime(new Date());
   }
 
+  useEffect(() => {
```

Comment:
I'd wrap the `LogsDatePicker` and the "Live" buttons together in a component that would be responsible for updating the time every second. I don't think we should re-render the whole header every second.

Basically lines 110-145 would become a new component with the state management included.